clients/stellar-relay-lib/src/connection/overlay_connection.rs

```diff
@@ -22,6 +22,15 @@ pub struct StellarOverlayConnection {
 	is_disconnected: bool,
 }
 
+impl StellarOverlayConnection {
+	pub fn get_actions_sender(&self) -> mpsc::Sender<ConnectorActions> {
+		self.actions_sender.clone()
+	}
+	pub fn disconnect_status(&self) -> ConnectorActions {
+		ConnectorActions::Disconnect
+	}
+}
+
```

Comment:
Let's maybe just move the two functions to the `impl` block below. But put it at the very bottom of it.

Reply:
Good point. Done.